djangocms_snippet/admin.py

```diff
@@ -18,11 +21,15 @@
 
 try:
     from djangocms_versioning.admin import ExtendedVersionAdminMixin
+
     if djangocms_versioning_enabled:
         snippet_admin_classes.insert(0, ExtendedVersionAdminMixin)
 except ImportError:
     djangocms_versioning_enabled = False
 
+TO_FIELD_VAR = '_to_field'
```

Comment:
These are defined by django, we should reuse that definition: 

from django.contrib.admin.options import TO_FIELD_VAR